test/docgen/view/__snapshots__/property.test.ts.snap

```diff
@@ -1,5 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`csharp snapshot 1`] = `
+" \`repositoryCatalogData\`<sup>Optional</sup> <a name=\\"Amazon.CDK.AWS.ECR.CfnPublicRepositoryProps.property.repositoryCatalogData\\"></a>
+
+\`\`\`csharp
+object repositoryCatalogData
```

Comment:
I wonder if there's something mode idiomatic here. Our AWS Dotnet docs show a getter like so:

```chsharp
public virtual string AppId { get; }
```

Is this appropriate here?